tests/e2e/standard/contacts/contact-summary.wdio-spec.js

```diff
@@ -0,0 +1,303 @@
+        name: 'national-user'
+      }
+    });
+  const districtAdminUser = userFactory.build(
+    {
+      username: 'user-district', place: placeBobClinic._id,
+      contact: {
+        _id: 'fixture:user-district:offline',
+        name: 'user-district'
+      },
+      roles: ['district_admin'],
+      known: false
+    });
+  const patientAlice = personFactory.build({ name: 'Alice Alison', phone: '+447765902001' });
+  const patientDavid = personFactory.build({ name: 'David Davidson', phone: '+447765902002', parent: placeBobClinic });
+  const davidVisit = reportFactory.build(
+    { form: 'visit' }, { patient: patientDavid, submitter: nationalAdminUser.contact });
+  const patientCarol = personFactory.build({
+    name: 'Carol Carolina', sex: 'f', parent: placeBobClinic, patient_id: '05946',
+    linked_docs: {
+      aliceTag: patientAlice._id,
+      davidTag: { _id: patientDavid._id },
+      visitTag: { _id: davidVisit._id },
+    }
+  });
+  const carolPregnancy = reportFactory.build(
+    { form: 'P' },
+    {
+      patient: patientCarol,
+      submitter: nationalAdminUser.contact
+    });
+  const carolVisit = reportFactory.build(
+    { form: 'V' },
+    {
+      patient: patientCarol,
+      submitter: nationalAdminUser.contact,
+      reported_date: moment().set('date', 5).valueOf(),
+      fields: {
+        visited_contact_uuid: placeBobClinic._id,
+        patient_id: patientCarol.patient_id
+      }
+    });
+
+  const docs = [patientAlice, placeBobClinic, patientCarol, patientDavid, carolPregnancy, davidVisit, carolVisit];
+
+  const settings = {
+    uhc: {
+      visit_count: {
+        month_start_date: 1,
+        visit_count_goal: 2
+      }
+    },
+    contact_summary: SCRIPT,
+    contact_types: [
+      {
+        id: 'clinic',
+        count_visits: true
+      },
+      {
+        id: 'person',
+        count_visits: false
+      }
+    ]
+  };
+
+  before(async () => {
+    await utils.saveDocs(docs);
+    await utils.createUsers([districtAdminUser, nationalAdminUser]);
+  });
+
+  afterEach(async () => {
+    await utils.revertSettings(true);
+    await commonElements.closeReloadModal();
+    await commonElements.logout();
+  });
+
+  after(async () => {
+    await utils.deleteUsers([nationalAdminUser, districtAdminUser]);
+    await utils.revertDb([/^form:/], true);
+  });
+
+  it('should load contact summary', async () => {
+    await loginPage.cookieLogin();
+    await commonElements.waitForPageLoaded();
+    await utils.updateSettings(settings, true);
+    await commonElements.closeReloadModal();
+    await commonElements.goToPeople();
+    await contactPage.selectLHSRowByText(patientCarol.name);
+    await contactPage.waitForContactLoaded();
+    // assert the summary card has the right fields
+
+    expect(await contactPage.cardFieldLabelText('test_pid')).to.equal('test_pid');
+    expect(await contactPage.cardFieldText('test_pid')).to.equal(patientCarol.patient_id);
```

Comment:
I see that these two asserts are repeated during all the tests several times.
Can you please create a separate method that will be in charge of asserting these two things? in that way, the test will look smaller and will be easier to read.

Can be something like this:

```.js
const validateCardField = async (text, value) => {
    expect(await contactPage.cardFieldLabelText(text)).to.equal(text);
    expect(await contactPage.cardFieldText(text)).to.equal(value);
};

await validateCardField('test_pid', patientCarol.patient_id);
```

Reply:
Thanks!